File: Core python/Assignment/Assignment 2/q7.py
# Find the sum of three-digit number.

# take 3 digit input number
n = int(input("enter three digit number : "))

# calculate last digit after iteration
d1 = n % 10
n = n // 10

d2 = n % 10
n = n // 10   # give as a last digit

d3 = n % 10

# calculate sum
sum = d1 + d2 + d3 

# print sum
print(f'sum of three-digit number is {sum} ')

File: C to Java/Assignments/Assignment 3/q7.c
/* Find factorial of given number.
Input: n = 5
Output: 120  */

void main() {
	
	int n ;
	printf("enter number : ");
	scanf("%d" , &n);
	
	int fact = 1 ;
	for(int i = 1 ; i <= n ; i++){
		fact = fact * i ;
	}
	
	printf("factorial is : %d", fact);
}